gossip/src/gossip_service.rs

```diff
@@ -54,6 +54,7 @@ impl GossipService {
             Recycler::default(),
             "gossip_receiver",
             1,
+            false,
```

Comment:
I thought that we need this to be pinned for signature verification using gpu, no?

Reply:
GPU doesn't support the gossip path today. We can enable it if we add it.